ansible/roles/host-ocp4-provisioner/tasks/main.yml

```diff
@@ -5,11 +5,11 @@
     path: /etc/ansible
     state: directory
 
-- when: cloud_provider == "osp"
-  import_tasks: "{{cloud_provider}}_prereqs.yml"
-
-- when: cloud_provider == "ec2"
-  import_tasks: "{{cloud_provider}}_prereqs.yml"
+- import_tasks: "{{cloud_provider}}_prereqs.yml"
```

Comment:
I don't think this logic works. Isn't a YAML list under when an "AND"? I think this needs to be cloud_provider == "osp" or cloud_provider=="ec2" ...

Reply:
fixed the logic, not sure what I was thinking.